### ISSUE TITLE
A command key that are not mentioned in the documentation: NVDA+control+f1

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1506]
**Reported by ateu on 2011-05-16 07:24**
Hello,
This is a very useful command key, but are not mentioned at the documentation. It is especially useful for developers.
NVDA+control+f1

**Comment by dkager** [comment]
Since users can't directly influence app modules except through their containing plug-ins I would also say the command can be removed. If it is to be kept it could be documented in the "Extra Tools" chapter.

Technical oddnes: for some reason NVDA+ctrl+f1 isn't properly registered on my PC. NVDA only registers NVDA+ctrl. Doing the same with other F-keys gives the expected result, as does pressing NVDA+alt+f1.

**Comment by Adriani90** [comment]
This command does nothing in 2018.2.1. I think this issue can be closed.

**Comment by dkager** [comment]
The command is still assigned by default:
`"kb:NVDA+control+f1": "reportAppModuleInfo",`

**Comment by netblue44** [comment]
It reads the name of the active aplcation and the NVDA modual that is 
loaded

**Comment by lukaszgo1** [comment]
@michaelDCurran @feerrenrut Are you planning to remove  this command as per [this commend](https://github.com/nvaccess/nvda/issues/1506#issuecomment-155290085). I am asking, because i am considering working on #7338 but if  this command is going to be removed soon i wouldn't waste my time.

**Comment by ehollig** [comment]
Looking at this issue, is NVDA+F1 documented somewhere? Should this command be documented in 12.1 in the user guide? How about the commands quick reference?

**Comment by feerrenrut** [comment]
@lukaszgo1 Sorry for the delay. My initial thought is to remove the default gesture, but keep the command. If so, then it should be documented along with an example use case.

Although, I think perhaps the question should be asked on the developers mailing list: Considering this information is available in devInfo, does anyone use this command directly, and why?

Would you be able to follow up on that for us?

**Comment by Brian1Gaff** [comment]
I find it interesting sometimes, for example if I'm in waterfox, it just 
says currently running application Waterfox.exe, but if as I am now in 
Outlook express, then it says Currently running application is msimn.exe and 
currently loaded module is msimn.
This presumably tells me there is no  app module for  waterfox.

I'm assuming therefore that as Firefox reports it has a firefox module that 
waterfox must be falling back to some native mozilla  code built in to nvda?

This may be important on the switch to python 3 if any of the older code 
suddenly does not work for some reason.
 Brian


Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
-----

**Comment by lukaszgo1** [comment]
@feerrenrut  wrote: 
> 
> 
> @lukaszgo1 Sorry for the delay.


No problem

>  My initial thought is to remove the default gesture, but keep the command. If so, then it should be documented along with an example use case.

I don't believe that is a good idea. While I can definitely understand adding new commands which don't have key command assigned removing existing one is very confusing and would be even more if something else would be assigned to it in the future.

> 
> Although, I think perhaps the question should be asked on the developers mailing list: Considering this information is available in devInfo, does anyone use this command directly, and why?
> 

I can ad  least present my own perspective.
This command is useful because:
1. Accessing the same information  from the dev info is certainly longer you have to read quite a bit of log to find it.
2. When this command would be removed it  would be required to move navigator object to the app for which you want to get this info, which for people like me working with obj nav not following the focus would decrease productivity.
3. Other screen eader's have such command therefore people are used to it.
4. For existing users removing anything breaks existing workflows and in my view should be avoided if possible.



In conclusion this command should remain assigned and be documented in my view.

**Comment by feerrenrut** [comment]
> While I can definitely understand adding new commands which don't have key command assigned removing existing one is very confusing and would be even more if something else would be assigned to it in the future

Generally I agree. In a case like this, where we think it is a fairly unused feature and those that use it are likely to know they need to re-assign the shortcut, then maybe it's ok?

I don't recommend we remove it at this stage. We should certainly document it, but we should still try to consider if we can remove the default shortcut.

**Comment by Adriani90** [comment]
I suggest removing the Default shortkey and create a new item in Input gestures Dialog where users can assign a shortcut for it. I think this Feature is important if we ask beginners who Report issues to deliver the Software Version in which they reproduce a specific issue. Often they do not know exactly where to look in the log file.

**Comment by lukaszgo1** [comment]
@Adriani90  wrote:
> 
> I think this Feature is important if we ask beginners who Report issues to deliver the Software Version in which they reproduce a specific issue. Often they do not know exactly where to look in the log file.


 
Currently this shortcut reports only an executable name and appmodule if present.

**Comment by lukaszgo1** [comment]
@feerrenrut  wrote:


> 
> I don't recommend we remove it at this stage. We should certainly document it, but we should still try to consider if we can remove the default shortcut.


May I ask why? In my view each change of functionality should give us something back. In this case however we aren't getting anything. As discussed above we cannon bind anything else by default to avoid confusion, and if for someone this combination would work better with a different script he/she can certainly reassign it himself already.
